Packs/SentinelOne/ReleaseNotes/4_0_0.md

```diff
@@ -0,0 +1,10 @@
+#### Integrations
```

Comment:
Please create the release note again using [demisto-sdk update-release-notes](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md) command